package.json

```diff
@@ -13,7 +13,9 @@
   "type": "module",
   "scripts": {
     "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js",
+    "test:q": "node --experimental-vm-modules node_modules/jest/bin/jest.js --silent",
     "test:cov": "node --experimental-vm-modules node_modules/jest/bin/jest.js --coverage --coverageReporters='text' --coverageReporters='text-summary'",
+    "test:cov:q": "node --experimental-vm-modules node_modules/jest/bin/jest.js --silent --coverage --coverageReporters='text' --coverageReporters='text-summary'",
     "lint": "eslint \"./**/*.js\" --cache --cache-strategy content --fix",
     "format": "prettier --write \"**/*.js\"",
     "format:njk": "prettier --write \"**/*.njk\" --ignore-path .prettierignore",
```

Comment:
Sorry to add on this line as I cannot add on line 23. The "install" script won't work as it will always run the husky before you manage to ignore scripts. There is no way AFAIK to ignore husky from scripts.

Reply:
OK, that's the line advised by @DavidBarker-EE which has been working for me. Can you offer a better alternative?